.stylelintrc.json

```diff
@@ -0,0 +1,10 @@
+{
+  "extends": ["stylelint-config-standard-scss", "stylelint-config-clean-order"],
+  "plugins": ["stylelint-selector-bem-pattern"],
+  "rules": {
+    "plugin/selector-bem-pattern": {
+      "preset": "bem"
+    },
```

Comment:
Not sure if this will require all selectors to be BEM, but it's not complaining about anything in `_header-bar.scss` for now